src/OpenTelemetry.Instrumentation.EntityFrameworkCore/EntityFrameworkInstrumentation.cs

```diff
@@ -21,6 +21,13 @@ namespace OpenTelemetry.Instrumentation.EntityFrameworkCore;
 
 internal class EntityFrameworkInstrumentation : IDisposable
 {
+    internal static readonly Func<object, object, string> DefaultDisplayNameFunc = (fetcher, connection)
+        =>
+    {
+        var propertyFetcher = (PropertyFetcher<object>)fetcher;
+        return (string)propertyFetcher.Fetch(connection);
+    };
```

Comment:
It is not used at all. It can be removed.

Reply:
forgot that, sorry